.travis.yml

```diff
@@ -1,27 +1,31 @@
 language: generic
 
-env:
-  matrix:
-    - EMACS=emacs24
-    - EMACS=emacs-snapshot
+before_install:
+  - curl -fsSkL https://gist.github.com/rejeep/ebcd57c3af83b049833b/raw > x.sh && source ./x.sh
```

Comment:
My spidey-sense says that its dangerous to pull from users gist and run it a a shell script.